py4syn/epics/DxpClass.py

```diff
@@ -104,25 +113,42 @@ def getValueChannel(self, **kwargs):
         channel = kwargs['channel']
         c = int(channel[3]) - 1
         if(len(channel) > 4):
-            r = int(channel[5])
-            return self.pvDxpRois[c][r]
+            r = int(channel[6])
```

Comment:
The PVs has the format mcaC.Rr (where C is channel and R is ROI). If PV is only mcaC then we call saveSpectrum, if is a mcaC.Rr, is just a point collect.

I change to a constant this values (4 and 6)